sshcode.go

```diff
@@ -608,3 +629,27 @@ func parseGCPSSHCmd(instance string) (ip, sshFlags string, err error) {
 
 	return strings.TrimSpace(userIP), sshFlags, nil
 }
+
+// When the user passes a path such as `/Workspace`, msys2 gives sshcode `C:/msys64/Worksapce`, which is not a valid remote path
+func gitbashWindowsDir(dir string) string {
+
+	// if dir is left empty, line82:main.go will set it to `~`, this makes it so that
+	// if dir is `~`, return `~/` instead of continuing with the gitbashWindowsDir()
+	// function. this prevens windows from trying to send a litteral instead of a relative
+	if dir == "~" {
+		return "~/"
+	}
+	mingwPrefix, _ := exec.Command("sh", "-c", "{ cd / && pwd -W; }").Output()
```

Comment:
```suggestion
	prefixBytes, err := exec.Command("sh", "-c", "{ cd / && pwd -W; }").Output()
	if err != nil {
		// Default to a sane location.
		mingwPrefix = []bytes("/mingw64")
	}
```

Are the `{}` around the command even needed? Also, could you please explain what `pwd -W` does? `-W` doesn't exist on Linux.

Reply:
`pwd -W` is a MinGW specific thing, I run `cd / && pwd -W` and instead of returning `/` like Linux and MinGW normally do, it will return the windows equivalent to that path, in this case, `C:/mingw64`

but yeah I will incorporate the fallback, bit of an oversight on my part

Reply:
![image](https://user-images.githubusercontent.com/10422110/78837670-69390580-79a9-11ea-9dd7-956e3d696e0c.png)